tests/sentry/issues/test_ingest_incident_integration.py

```diff
@@ -232,3 +238,70 @@ def test_can_resolve_outstanding_incident(self) -> None:
         assert last_activity_entry.type == 2
         assert last_activity_entry.previous_value == str(IncidentStatus.CRITICAL.value)
         assert last_activity_entry.value == str(IncidentStatus.CLOSED.value)
+
+    @mock.patch("sentry.models.group.logger")
+    def test_bulk_transition_new_group_to_ongoing__metric_issues__no_incident_activites(
+        self, mock_logger
+    ) -> None:
+        """Don't want to trigger an IGOP write for this status change"""
+        _, group_info = self.save_issue_occurrence()
+        group = group_info.group
+        assert group is not None
+
+        assert GroupOpenPeriod.objects.filter(group=group, project=self.project).exists()
+
+        open_period = GroupOpenPeriod.objects.get(group=group, project=self.project)
```

Comment:
Maybe something like this to avoid 2 queries would do the same thing

```suggestion
        open_period = GroupOpenPeriod.objects.get(group=group, project=self.project)
        assert open_period
```